package.json

```diff
@@ -1,30 +1,33 @@
 {
   "name": "openui5-sample-app",
-  "version": "0.1.0",
+  "version": "0.2.0",
   "description": "Sample of an OpenUI5 app",
   "private": true,
   "scripts": {
-    "test": "grunt lint && grunt test",
-    "start": "grunt serve",
-    "build": "grunt build",
-    "postinstall": "bower install"
+    "start": "ui5 serve",
+    "lint": "eslint webapp",
+    "karma": "karma start",
+    "karma-ci": "karma start karma-ci.conf.js",
+    "watch": "start-server-and-test start http://localhost:8080 karma",
+    "test": "npm run lint && rimraf coverage && start-server-and-test start http://localhost:8080 karma-ci",
+    "build": "rimraf dist && ui5 build --a"
+  },
+  "dependencies": {
+    "@openui5/sap.m": "^1.52.5",
+    "@openui5/sap.ui.core": "^1.52.5",
+    "@openui5/themelib_sap_belize": "^1.52.5"
   },
   "devDependencies": {
-    "bower": "^1.8.0",
-    "grunt": "^1.0.1",
-    "grunt-contrib-clean": "^1.0.0",
-    "grunt-contrib-connect": "^1.0.2",
-    "grunt-contrib-copy": "^1.0.0",
-    "grunt-eslint": "^20.0.0",
-    "grunt-karma": "^1.0.0",
-    "grunt-openui5": "^0.12.0",
-    "karma": "^2.0.0",
+    "@ui5/cli": "0.0.1",
```

Comment:
Should we add this here or treat it as a general prerequisite (like grunt-cli in the past)?

And if we add it here, why not ranged: `^0.0.1`?